core/node/state_keeper/src/batch_executor/tests/tester.rs

```diff
@@ -356,17 +356,17 @@ impl AccountLoadNextExecutable for Account {
 
     /// Returns a transaction to the loadnext contract with custom amount of write requests.
     /// Increments the account nonce.
-    fn loadnext_custom_writes_call(
+    fn loadnext_custom_new_writes_call(
         &mut self,
         address: Address,
-        writes: u32,
+        new_writes: u32,
         gas_limit: u32,
     ) -> Transaction {
         // For each iteration of the expensive contract, there are two slots that are updated:
         // the length of the vector and the new slot with the element itself.
         let minimal_fee = 2
             * testonly::DEFAULT_GAS_PER_PUBDATA
-            * writes
+            * new_writes
             * INITIAL_STORAGE_WRITE_PUBDATA_BYTES as u32;
```

Comment:
Does this fee calculation need to change based on the introduction of `over_writes` as well?